auth/grpctransport/grpctransport.go

```diff
@@ -151,6 +154,8 @@ type InternalOptions struct {
 	// DefaultScopes specifies the default OAuth2 scopes to be used for a
 	// service.
 	DefaultScopes []string
+	// SkipValidation bypasses validation on Options.
+	SkipValidation bool
```

Comment:
Can we include a little bit about when or why we'd use this? Seems a little bit like a big red button :)

Reply:
Added a little more. Note it is in the InternalOptions struct which we don't want to encourge users to use. Corresponds to https://pkg.go.dev/google.golang.org/api/option/internaloption#SkipDialSettingsValidation.

Reply:
Yep totally, I just personally wasn't sure what this was for so the comment & your link helps, thanks.